### PR TITLE
expose manipulatorMouseWrap to settings

### PR DESCRIPTION
**Pull request by amzn-ahmadkrm**
*Added manipulatorMouseWrap to settings from existing functionality in ViewportManipulatorController.cpp

Duplicate PR due to the first one being against the wrong target branch 

Signed-off-by: amzn-ahmadkrm [ahmadkrm@amazon.com](mailto:ahmadkrm@amazon.com)

### CODE REVIEW

**Review by hultonha — APPROVED** [review]
LGTM! 😄 Nice work thanks 👍

**Comment by hultonha** [comment]
Adding @pollend for visibility too as they worked on this initial feature

File: Code/Editor/EditorPreferencesPageViewportManipulator.cpp [hunk 1]
```diff
@@ -96,7 +97,10 @@ void CEditorPreferencesPage_ViewportManipulator::Reflect(AZ::SerializeContext& s
             ->Attribute(AZ::Edit::Attributes::Max, AzToolsFramework::MaxManipulatorViewBaseScale)
             ->DataElement(
                 AZ::Edit::UIHandlers::CheckBox, &Manipulators::m_flipManipulatorAxesTowardsView, "Flip Manipulator Axes Towards View",
-                "Determines whether Planar and Linear Manipulators should switch to face the view (camera) in the Editor");
+                "Determines whether Planar and Linear Manipulators should switch to face the view (camera) in the Editor")
+            ->DataElement(
+                AZ::Edit::UIHandlers::CheckBox, &Manipulators::m_manipulatorMouseWrap, "Manipulator Mouse Wrap",
+                "Determines whether the cursor is going to wrap around the source widget.");
```
[the remark this answers is not in the document]
Comment (hultonha) [reply Code/Editor/EditorPreferencesPageViewportManipulator.cpp:103]: Could go for 'viewport'?

**Review by pollend — APPROVED** [review]

**Review by moraaar — APPROVED** [review]

**Comment by hultonha** [comment]
This is not a change we're going to include right now, closing this PR.

**Pull request closed without merging**